pip/pip-274.md

```diff
@@ -0,0 +1,65 @@
+# Motivation
+
+The metrics are all started with `pulsar_`, so that both users and operators can quickly find the metrics of the entire system through this prefix. However, due to some other reasons, 
+it was found that `topic_load_times` was missing the prefix, so want to get it right.
+
+# Goals
+
+## In Scope
+
+### Proposed changes
+
+In master branch, keep the old metric `topic_load_times` and add below new metrics:
+
+* `pulsar_topic_load_times`
+
+After release-3.1.0, remove ``topic_load_times`.
+
+## Out of Scope
+
+
+
+# High Level Design
+
+
+
+# Detailed Design
+
+## Design & Implementation Details
+
+
+
+## Public-facing Changes
+
+
+
+### Public API
```

Comment:
Feel free to delete any title with empty body

Reply:
fixed